python/hail/api2/table.py

```diff
@@ -1079,6 +1079,20 @@ def show(self, n=10, width=90, truncate=None, types=True):
             |     4 |    60 | F      |     8 |     2 |    11 |    90 |   -10 |
             +-------+-------+--------+-------+-------+-------+-------+-------+
 
+        Notes
+        -----
+        This method accepts print configuration parameters as keyword
+        arguments. The valid parameters are:
+
+         - ``missing``: The literal string to be printed for missing values.
+           The default is ``'NA'``.
+         - ``float_format``: The format string to be used for floating-point
+           values. The default is ``%.4e``, which prints scientific notation
+           with four significant figures after the decimal point.
```

Comment:
I think we should link to the definition of our format strings: [`java.util.Formatter`](https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html)